en/assembly/mount_gps_compass.md

```diff
@@ -7,13 +7,25 @@ The diagram below shows the heading marker on the Pixhawk 4 and compass.
 
 ## Compass Orientation
 
-The compass can also be mounted in any other _supported_ orientation, where the supported orientations can be seen in [CAL_MAGn_ROT](../advanced_config/parameter_reference.md#CAL_MAG1_ROT), and have the same meaning as used for [orienting the flight controller](../config/flight_controller_orientation.md#calculating-orientation).
+The compass can also be mounted in any _supported_ orientation, where the supported orientations can be seen in [CAL_MAGn_ROT](../advanced_config/parameter_reference.md#CAL_MAG1_ROT), and uses the same convention for [orienting the flight controller](../config/flight_controller_orientation.md#orientation-definition).
 
 If you're using the normal [Compass Calibration](../config/compass.md) process the parameter [CAL_MAG_ROT_AUTO](../advanced_config/parameter_reference.md#CAL_MAG_ROT_AUTO) is enabled, and the orientation should be detected automatically.
 
-:::warning
-You must mount the compass in a supported orientation!
+If your compass can't be oriented in any of the supported orientations, you can utilize `Custom rotation`, which is documented below.
 
-If you mount the compass at an orientation that isn't supported, for example `Yaw 30`, PX4 will detect the closest supported value.
-This will result in errors/warnings, even if the calibration appeared to succeed.
+:::warning
+If you mount the compass at an orientation that isn't supported, for example `Yaw 30`, PX4 will detect the closest supported value. This will result in errors/warnings, even if the calibration appeared to succeed.
```

Comment:
```suggestion
If you mount the compass at an orientation that isn't supported, for example `Yaw 30`, PX4 will detect the closest supported value.
This may result in errors/warnings, even if the calibration appeared to succeed.
```

Reply:
Yes! I also thought this part isn't so necessary :+1: